examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/ui/screens/paywall/PaywallScreenViewModel.kt

```diff
@@ -39,13 +39,45 @@ class PaywallScreenViewModelImpl(
         updateOffering()
     }
 
+    override fun onRestoreStarted() {
+        val value = _state.value
+        if (value is PaywallScreenState.Loaded) {
+            _state.update {
+                value.copy(
+                    dialogText = "Restoring purchases...",
+                )
+            }
+        }
+    }
+
+    override fun onRestoreCompleted(customerInfo: CustomerInfo) {
+        val value = _state.value
+        if (value is PaywallScreenState.Loaded) {
+            _state.update {
+                value.copy(
+                    dialogText = "Restore completed",
+                )
+            }
+        }
+    }
+
+    override fun onRestoreError(error: PurchasesError) {
+        val value = _state.value
+        if (value is PaywallScreenState.Loaded) {
+            _state.update {
+                value.copy(
+                    dialogText = "There was an error restoring purchases",
```

Comment:
I think it might be worth displaying the error message in the alert... I understand we wouldn't do that in a user-facing app, but it might be helpful as a tester app... Wdyt?

Reply:
sounds good! going to merge this into the other PR and make the change there